analysis/analyze-hotspots/build.gradle

```
@@ -36,6 +36,18 @@ dependencies {
     natives "com.esri.arcgisruntime:arcgis-java-resources:$arcgisVersion"
 }
 
+task apiKey {
+    description = "Creates a new gradle.properties file with an empty API key variable in the user home ./gradle folder if the file doesn't already exist."
```

I think another comma here would be good to break the sentence up:
```suggestion
    description = "Creates a new gradle.properties file with an empty API key variable in the user home ./gradle folder, if the file doesn't already exist."
```